src/design/place.jl

```diff
@@ -85,7 +85,7 @@ function place{S<:Real,U<:Real,V<:Number}(
   d = Poleplacement(A, B, p; α=α, rtol=rtol)
 
   if(size(A,1) - d.m  > 0) # if there is something to optimize over
-    G0 = randn(eltype(A), size(A,1), size(B,2)) # random initial point
```

Comment:
This did not function properly when A has integer elements. The initial guess is now always in float.